front/src/modules/ui/input/double-text/components/DoubleTextInputEdit.tsx

```diff
@@ -33,24 +41,34 @@ export function DoubleTextInputEdit({
 }: DoubleTextInputEditProps) {
   return (
     <StyledDoubleTextContainer>
-      <StyledNameInput
-        size={firstValue.length}
-        autoFocus
-        placeholder={firstValuePlaceholder}
-        value={firstValue}
-        onChange={(event: ChangeEvent<HTMLInputElement>) => {
-          onChange(event.target.value, secondValue);
-        }}
-      />
-      <StyledNameInput
-        size={secondValue.length}
-        autoComplete="off"
-        placeholder={secondValuePlaceholder}
-        value={secondValue}
-        onChange={(event: ChangeEvent<HTMLInputElement>) => {
-          onChange(firstValue, event.target.value);
-        }}
-      />
+      <TemplateDimensionsEffect template={firstValue || firstValuePlaceholder}>
```

Comment:
I like the API but I think that "Template" keyword is not clear
What about: `ComputeNodeDimensions` and `node`?